R/provider-anthropic.R

```diff
@@ -17,6 +17,17 @@ NULL
 #' @inherit chat_openai return
 #' @param model `r param_model("claude-sonnet-4-20250514", "anthropic")`
 #' @param api_key `r api_key_param("ANTHROPIC_API_KEY")`
+#' @param cache How long to cache inputs? Defaults to "5m" (five minutes).
+#'   Set to "none" to disable caching or "1h" to cache for one hour.
+#'
+#'   Caching (for five minutes) is enabled by default because it will save you
+#'   money for multi-turn conversations. Cache writes cost 125%, but cache
```

Comment:
Probably worth mentioning that the 5m cache costs 1.25 x input tokens while the 1h cache costs 2x.